client/src/components/AddCohort/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './style.css';
+import { Form, Input, Button, message } from 'antd';
+import postData from '../postData';
+
+const Demo = () => {
```

Comment:
I'd suggest using a more descriptive name, like `AddCohortForm` for example.  This name is used in react devtools in the component hierarchy view, so it can be helpful to have a descriptive name for there.